src/app/auth/singup/singup.component.ts

```diff
@@ -8,7 +8,7 @@ import { AuthenticateService } from 'src/app/authenticate.service';
   templateUrl: './singup.component.html',
   styleUrls: ['./singup.component.css'],
 })
-export class SingupComponent implements OnInit {
+export class SingUpComponent implements OnInit {
```

Comment:
Could you fix the spelling on this?
SignUp rather than SingUp

Reply:
updated